src/app/layouts/components/layout-wrapper/layout-wrapper.component.scss

```
@@ -15,6 +15,39 @@
+     * the drawer's `transitionend` event to emit `openedChange` and update the content margins /
+     * focus trap. A keyframe animation would fire `animationend` instead and break that contract.
+     */
+    .mat-drawer.menu-sidenav {
+      will-change: transform;
+
+      &:not(.mat-drawer-opened):not(.mat-drawer-animating) {
+        visibility: visible;
+
+        // `.mat-drawer-inner-container` belongs to Material's own template, so it needs
+        // `::ng-deep` to be reached from this emulated-encapsulation stylesheet.
+        ::ng-deep .mat-drawer-inner-container {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)